scout/adapter/mongo/variant.py

```diff
@@ -326,6 +339,32 @@ def gene_variants(
 
         return result
 
+    def count_gene_variants(
+        self, query=None, category="snv", variant_type=["clinical"], institute_id=None
+    ):
+        """Count all variants seen in a given gene.
+
+        Arguments:
```

Comment:
It's OK but to be consistent with the other docstrings, could you rename this one to Args and add a Returns?

Reply:
This is inconsistent all over the code base. "Args:" win 381 to 58 for "Arguments". I could to a separate refactoring PR and change it everywhere. 

Though, personally I think "Arguments" is nicer.

ps. Looking into Python source code the ratio is about same. Args: 1129, Arguments: 211.

Reply:
Bah I've always used Args, and as you counted it's mostly Args, so I'd change it to Args. If you want to rename it all over the place is fine, but I don't think it's super important at the moment

Reply:
[Type hints!!!](https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html) then we can remove all of those

Reply:
I agree, but sometimes is also good to have some more info from the arguments instead of only the expected type..

Reply:
Done!